docs/how-to/redeploy.md

```diff
@@ -0,0 +1,19 @@
+# How to redeploy Jenkins
+
+We consider a redeployment is the process where the old charm instance is removed and data is migrated to a new charm instance.
```

Comment:
Nit: "Redeployment is a process where the old charm instance is removed and data is migrated to a new charm instance."

Reply:
Thanks, that was my 2 lines of thought joining and I forgot to clean them up.

Reply:
I updated it